desktop/src/main/java/bisq/desktop/components/paymentmethods/BankForm.java

```diff
@@ -240,28 +240,28 @@ public void addFormForAddAccount() {
         bankNameInputTextField = addInputTextField(gridPane, ++gridRow, Res.get("payment.bank.name"));
 
         bankNameInputTextField.textProperty().addListener((ov, oldValue, newValue) -> {
-            bankAccountPayload.setBankName(newValue);
+            bankAccountPayload.setBankName(newValue.trim());
```

Comment:
I wonder if we should include this functionality to the `BankAccountPayload` class itself. @ripcurlx what is your opinion?

Reply:
I think the Payload objects should be POJOs without validation or verification code IMO. In that case the BankForm is an abstract class anyways and will be reused by all BankForm implementations.